components/PlantCard/index.tsx

```diff
@@ -1,43 +1,60 @@
 import React from 'react';
 import { Plant } from '@/types/schema';
-import styles from './PlantCardStyles.module.css';
+import {
+  Attribute,
+  Card,
+  CardContent,
+  CardPic,
+  PlantAttributes,
+  RoundCheck,
+  TopRight,
+} from './styles';
 
 export default function PlantCard({
   plant,
   canSelect,
+  isSelected,
+  onToggleSelect,
```

Comment:
1. what if we changed `onToggleSelect` to `onClick`? or 
2. what if we added `onClick` as another prop? then on line 27, we could do `onClick={canSelect ? onToggleSelect : onClick}`